internal/compiler/widgets/qt/combobox.slint

```diff
@@ -2,58 +2,64 @@
 // SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
 
 import { ComboBoxBase } from "../common/combobox-base.slint";
+import { ListView } from "../common/listview.slint";
 
 export component ComboBox {
-    in property <[string]> model <=> i-base.model;
-    in property <bool> enabled <=> i-base.enabled;
-    out property <bool> has-focus <=> i-base.has-focus;
-    in-out property <int> current-index <=> i-base.current-index;
-    in-out property <string> current-value <=> i-base.current-value;
+    in property <[string]> model <=> base.model;
+    in property <bool> enabled <=> base.enabled;
+    out property <bool> has-focus <=> base.has-focus;
+    in-out property <int> current-index <=> base.current-index;
+    in-out property <string> current-value <=> base.current-value;
 
-    callback selected <=> i-base.selected;
+    callback selected <=> base.selected;
+    
+    property <length> item-height: 40px;
 
     accessible-role: combobox;
     accessible-value <=> root.current-value;
-    forward-focus: i-base;
+    forward-focus: base;
 
     HorizontalLayout {
-        i-native := NativeComboBox {
+        native := NativeComboBox {
             current-value <=> root.current-value;
             has-focus <=> root.has-focus;
             enabled <=> root.enabled;
         }
     }
 
-    i-base := ComboBoxBase {
+    base := ComboBoxBase {
         width: 100%;
         height: 100%;
         show-popup => {
-            i-popup.show();
+            popup.show();
         }
     }
 
-    i-popup := PopupWindow {
+    popup := PopupWindow {
         x: 0;
         y: root.height;
         width: root.width;
+        height: base.popup-height(5, root.item-height);
 
         NativeComboBoxPopup {
             width: 100%;
             height: 100%;
         }
 
-        VerticalLayout {
-            spacing: 0px;
+        ListView {
+            width: parent.width;
+            height: parent.height;
 
             for value[index] in root.model: NativeStandardListViewItem {
                 item: { text: value };
                 is-selected: root.current-index == index;
                 has-hover: ta.has-hover;
                 combobox: true;
+                height: root.item-height;
```

Comment:
Even if we hardcode the size for the purpose of computing the maximum height, we shouldn't hardcode the size here.